### PR TITLE
added a "Star us on GitHub" button: (This PR resolves Issue #6)

### PR DESCRIPTION
**Pull request by SCR01**
### Description
This PR adds a "Star us on GitHub" button to the `Hero` component to allow users to star the repository and drive engagement quickly.

---

### Changes
- A button was added at the top right corner of the `Hero` component.
- The button includes the Star logo and text, "Star us on GitHub".
- The button links to the GitFinder repository on GitHub.

---

### How to Test
1. Run the application.
2. Verify the "Star us on GitHub" button is visible in the top right corner.
3. Click the button to ensure it redirects to the GitFinder GitHub repository.

Screenshot 2024-10-02 031915

### CODE REVIEW

File: src/firebase.js [hunk 1]
```diff
@@ -6,13 +6,13 @@ import "firebase/compat/firestore";
 import "firebase/compat/database";
 
 const firebaseConfig = {
-  apiKey: process.env.REACT_APP_API_KEY,
```
Comment (BamaCharanChhandogi) [inline src/firebase.js:9]: Why did you change this file?

Comment (SCR01) [reply src/firebase.js:9]: @BamaCharanChhandogi I did this because it is not working locally in my system. It is having some uncaught runtime error.

**Comment by SCR01** [comment]
@BamaCharanChhandogi should I redo it or it is fine?

File: src/firebase.js (hunk 1 above)
Comment (yasheela-alla) [reply src/firebase.js:9]: yeah same problem, i created a seperate firebase config, still not working

**Comment by SCR01** [comment]
@BamaCharanChhandogi Hey, Can you please merge it now!

File: src/firebase.js (hunk 1 above)
Comment (BamaCharanChhandogi) [reply src/firebase.js:9]: I have added a .env.example file. Tell me if any issues are coming up or not!

**Comment by BamaCharanChhandogi** [comment]
I have added a .env.example file. Tell me if any issues are coming up or not!

**Comment by BamaCharanChhandogi** [comment]
> @BamaCharanChhandogi should I redo it or it is fine?

You have to redo

File: src/firebase.js (hunk 1 above)
Comment (SCR01) [reply src/firebase.js:9]: @BamaCharanChhandogi Yeah it is working fine

**Comment by SCR01** [comment]
Screenshot 2024-10-04 073935

looking good at my end!

**Comment by SCR01** [comment]
@BamaCharanChhandogi I redo the changes, Kindly check it.

**Review by BamaCharanChhandogi — APPROVED** [review]
LGTM,thanks
